src/NetDaemon.Templates.Project/templates/netdaemon_src/.gitignore

```diff
@@ -1,6 +1,8 @@
 obj
 bin
-#appsettings.json
+appsettings.Development.json
 .vs
 *.gen
 .idea
+#appsettings.json
+#HomeAssistantGenerated.cs
```

Comment:
Is this in comment? Why is that? I do like to commit it because it is actually needed to compile the code

Reply:
It is comment out cause most people do want to commit those I think.  Since it is commented it will be committed as default. If they do not want to commit they will just remove the comments.